data/sql/updates/pending_db_world/rev_1599321019527522400.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1599321019527522400');
+UPDATE creature_template SET modelid1='30583' WHERE (entry='32315');
```

Comment:
No.
Update the modelID in `creature` not `creature_template`.

Reply:
I updated it

Reply:
please don't use strings where you can just use numbers:

```sql
UPDATE creature_template SET modelid1 = 30583 WHERE entry = 32315;
```

Reply:
@Akimboss please resolve @Kitzunu 's comment

Reply:
I still stand by my comment here.